app/views/handlers/__init__.py

```diff
@@ -38,3 +39,12 @@ def individual_response_url(
                 journey=journey,
             )
     return None
+
+
+def show_individual_response_link(location, routing_path):
+    if (
+        location.block_id == routing_path[0]
+        and location.list_item_id != "primary-person"
```

Comment:
`lisT_item_id` Is never `primary-person`.

Reply:
Sorry, I was checking something and left i there..